### ISSUE TITLE
where the requirement.txt?

### ISSUE THREAD

**Issue by bilibilicharger** [ispc-lab/HRegNet2#2]
as title

**Comment by FanLu97** [comment]
Sorry, the current status of this repo is not final version and we need some time for further updating.

**Comment by bilibilicharger** [comment]
> Sorry, the current status of this repo is not final version and we need some time for further updating.

ok，I ran the code successfully and without an error and with a result, I don't quite know if there's something wrong with this code

**Comment by FanLu97** [comment]
Yes, you can run the code successfully. However, the current implementation is an early version of this work and is not fully consistent with the latest paper. Some modules have not been integrated into this repo.

**Comment by Wansit99** [comment]
> Yes, you can run the code successfully. However, the current implementation is an early version of this work and is not fully consistent with the latest paper. Some modules have not been integrated into this repo.

Thank for your greate work! But when will there be an update?

**Comment by Nasrudin78** [comment]
Thanks for your contribution. Any update on making code consistent with paper?
Regards

**Comment by rasrab1992** [comment]
Thanks for your great work. Any update on making code consistent with paper?
Best Regards
